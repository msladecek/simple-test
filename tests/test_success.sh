#!/usr/bin/env sh
./simple-test examples/success/*.py
