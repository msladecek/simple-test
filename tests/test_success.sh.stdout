examples/success/test_everything.py SUCCESS
examples/success/test_exit_code.py SUCCESS
examples/success/test_hello_world.py SUCCESS
examples/success/test_input_stream.py SUCCESS
examples/success/test_no_shebang.py SUCCESS
examples/success/test_output_streams.py SUCCESS
